### PR TITLE
Add Use Cases, Target user profile, value proposition, and user stories documentation

### PR DESCRIPTION
**Pull request by alphajae11**
Developer Guide has been edited to include more details about the project - CCA Manager.

Add Use Cases, Target user profile, value proposition, and user stories documentation of this project. All of the details are documented until version 1.2. The above information for the remaining features that will be made in later versions will be updated to the Developer Guide later.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -274,56 +274,109 @@ _{Explain here how the data archiving feature will be implemented}_
...
+* 4a. Invalid Group Name
+
+  Use case ends.
+
+* 5a. Invalid Index of Contacts
+
+  Use case ends.
+
+
+UC03 )Identify the EXCO members
```
Comment (JuliaPoo) [inline docs/DeveloperGuide.md:372]: `UC03 )` -> `UC03) `

**Review by JuliaPoo — CHANGES_REQUESTED** [review]
Yea looks good apart from that

**Review by JuliaPoo — APPROVED** [review]

**Comment by JuliaPoo** [comment]
Actually if you don't mind I'll do some formatting to this branch?
In particular:
- Standardise (and inline-code) glossary terms
- Inline-code verbatim texts
- Use markdown's enumerations (instead of `2a)` etc)

**Comment by JuliaPoo** [comment]
Fixes #10.

**Comment by alphajae11** [comment]
fixes #10 #9

**Pull request merged**